app/Bus/Handlers/Commands/Subscriber/UpdateSubscriberCommandHandler.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of Cachet.
+ *
+ * (c) Alt Three Services Limited
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace CachetHQ\Cachet\Bus\Handlers\Commands\Subscriber;
+
+use CachetHQ\Cachet\Bus\Commands\Subscriber\UpdateSubscriberCommand;
+use CachetHQ\Cachet\Bus\Commands\Subscriber\UpdateSubscriberSubscriptionCommand;
+use Carbon\Carbon;
+
+/**
+ * This is the subscribe subscriber command handler.
+ *
+ * @author Piotr Grabowski <pgrabowski@fandom.com>
+ */
+class UpdateSubscriberCommandHandler
+{
+    /**
+     * Handle the update subscriber command.
+     *
+     * @param \CachetHQ\Cachet\Bus\Commands\Subscriber\UpdateSubscriberCommand $command
+     *
+     * @return \CachetHQ\Cachet\Models\Subscriber
+     */
+    public function handle(UpdateSubscriberCommand $command)
+    {
+        $subscriber = $command->subscriber;
+
+        $subscriber->email = $command->email;
+
+        if (!$subscriber->is_verified && $command->verified) {
```

Comment:
hmm... not really sure how I can tidy it up.
A Subscriber should get a `verified_at` only if they were not verified before, and an Admin selected the checkbox `verify`.
I removed the `elseif`. I am open to suggestions on how to improve it :)